<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>651</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Evenement</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>651</width>
        <height>341</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_ajouter_emp">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLabel" name="label_id_c">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>CODE</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_cin_c">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_nom_c">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>110</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_prenom_c">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>150</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_code">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_date">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>110</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_type">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajouter_E">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_quitter">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_modifier_E">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_rafraichir_E">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Rafraichir</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>190</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Ville</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>230</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Prix</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ville">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>190</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prix">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>230</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_afficher_emp">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>531</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supprimer_E">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>40</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_rafraichir">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Rafraichir</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2_quitter">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_tri_codee">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri code</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_tri_nom">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri nom</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_tri_prix">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri prix</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_chercher_type">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_chercher_ville">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>280</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_chercher_type">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>280</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Chercher type</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_chercher_ville">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Chercher ville</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_pdf">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>70</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Statistique</string>
       </attribute>
       <widget class="QCustomPlot" name="customPlot" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>621</width>
          <height>261</height>
         </rect>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_statistique">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>10</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Afficher Statistique</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
